docs/UserGuide.md

```diff
@@ -184,7 +183,7 @@ Format: `tag INDEX -t TAG_NAME`
 * Tags `TAG_NAME` for food item at the specified `INDEX`.
 * `TAG_NAME` is case-sensitive.
 * The index refers to the index number shown in the displayed food list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* The index **must be a positive integer** 1, 2, 3, …​ that is smaller than 2^31
```

Comment:
It should be no larger than the size of the list instead of smaller than 2^31

Reply:
For this, assume that the user inputs some number that is larger MAX_INT. What message should we return in that case: the index is larger than 2^31, or the index is larger than the size of the list?

Reply:
hmm, then it should be the minimum of the 2?